web/src/components/meetingStationPicker/MeetingStationPicker.jsx

```diff
@@ -115,6 +118,7 @@ const MeetingStationPicker = ({
 										placeholder=" "
 										pattern={stationRegex}
 										onInvalid={(e) => handleInvalidInput(e, index)}
+										onBlur={(e) => handleInvalidInput(e, index)}
```

Comment:
I'm unhappy with the naming here.  It's not "handleInvalidInput" is it? It's "validateInput" I think?